Source/Core/Common/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ add_library(common
   ENet.h
   EnumFormatter.h
   EnumMap.h
+  EnumUtils.h
```

Comment:
Missing the VS project change.

Reply:
It's strange, I don't remember this ever being a step when adding headers in PRs in the past.

Reply:
Is it just to the DolphinLib.props file?

Reply:
Yes.